src/rdo.rs

```diff
@@ -603,7 +593,64 @@ pub fn rdo_mode_decision(
   });
 
   if !best.skip {
-    intra_mode_set.iter().for_each(|&luma_mode| {
+    let tx_size = match bsize {
```

Comment:
Here I believe you can use `bsize.tx_size()` to avoid this large block.

Reply:
Good point